exporter/elasticsearchexporter/factory.go

```diff
@@ -93,11 +95,11 @@ func createDefaultConfig() component.Config {
 		},
 		Batcher: BatcherConfig{
 			FlushTimeout: 30 * time.Second,
-			MinSizeConfig: exporterbatcher.MinSizeConfig{
-				MinSizeItems: 5000,
+			MinSizeConfig: exporterbatcher.MinSizeConfig{ //nolint:staticcheck
```

Comment:
cc @carsonip, @lahsivjar, @axw. I'v'e added the nolint pragmas instead of migrating to the new interface because this exporter has its own `BatcherConfig`. Please address this separately. See https://github.com/open-telemetry/opentelemetry-collector/issues/12303 and https://github.com/open-telemetry/opentelemetry-collector/pull/12486

Reply:
Thanks for the ping, will look into it soon.